duke-core/src/test/java/no/priv/garshol/duke/cleaners/StopwordsCleanerTest.java

```diff
@@ -0,0 +1,21 @@
+
+package no.priv.garshol.duke.cleaners;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+
+public class StopwordsCleanerTest extends LowerCaseNormalizeCleanerTest {
+
+    public void setUp() {
+        cleaner = new StopwordsCleaner();
+    }
+
+    public void testMapping() {
+        assertEquals("Hello my name is duke", cleaner.clean("hello name duke"));
```

Comment:
This test looks wrong. Surely the two string literals should be swapped?

Please also add two more tests: one with null and one with "". We need to know the cleaner can handle these two cases.